charts/pega/charts/constellation/templates/clln-deployment.yaml

```diff
@@ -0,0 +1,36 @@
+#Deploy only when the constellation flag has been enabled in the values yaml.
+{{ if and .Values.enabled (eq .Values.enabled true) }}
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: constellation
+  labels:
+    app: constellation
+    # component: constellation
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: constellation
+      # component: constellation
+  template:
+    metadata:
+      labels:
+        app: constellation
+        # component: constellation
+    spec:
+      imagePullSecrets:
+      - name: regcred
```

Comment:
This should use the registry secret that is included with the platform (See https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/templates/_pega-deployment.tpl#L178)

Reply:
Thank you @dcasavant for your input. Please check all comments and let us know your thoughts, post we will create new PR.
Sure we will remove and use {{ template "pegaRegistrySecret" }}